server/db/db.js

```diff
@@ -2,19 +2,78 @@ const Pool = require('pg').Pool
 const pool = new Pool({
   user: 'austinchapin',
   host: 'localhost',
-  database: 'test',
+  database: 'ratings_reviews',
```

Comment:
I used process.env variables here, you can create a .env file, set USER="austinchapin" within it, import with require('dotenv').config() at the top of this module, and access with process.env.USER. I don't think it matters here, but probably a good practice.